remove_item.php

```diff
@@ -0,0 +1,50 @@
+<?php
+  session_start();
+  include_once('check_session.php');
+  include_once('database.php');
+
+  // TODO: Import database funcs from here
+  // include_once('database_funcs.php');
+
+  if (isset($_POST['type']) && $_POST['type'] === 'remove') {
+      remove_item();
+  }
+
+  // TODO: Move this to database_funcs.php
+  function remove_item() {
+      global $connection;
+
+      $user_id = $_SESSION['id'];
+      if (isset($_POST['item_id'])) {
+          $item_id = $_POST['item_id'];
+
+          $get_user_id = "SELECT user_id FROM Item WHERE id = $item_id";
+          $user_id_result = mysqli_query($connection, $get_user_id);
+
+          if (!$user_id_result) {
+              echo json_encode(['status' => 'error', 'message' => 'No such user']);
+              exit();
+          }
+
+          $row_user = mysqli_fetch_array($user_id_result);
+
+          if (!$row_user || $row_user[0] !== $user_id) {
+              echo json_encode(['status' => 'error', 'message' => 'Unauthorized process']);
+              exit();
+          }
+
+          // Update SQL query to update the item
+          $update_query = "UPDATE Item SET is_available = 0 WHERE id = $item_id";
```

Comment:
This would still keep the item in the table. Should we not be dropping the row instead / deleting the data?
is_available only indicates if the item can be added to an auction or not since we want to avoid the situation where an item is added to 2 auctions and then sold in one.
If we want the ability to restore it, we can add another attribute for is_deleted or something.

Would love views @paulinalydav @gazalguptaa @jasmehr

Reply:
Okay that makes sense, resolving comment